tests/test_app/components.py

```diff
@@ -1,22 +1,25 @@
 import idom
+from idom import html
+from test_app.models import TodoItem
 
 import django_idom
+from django_idom.hooks import use_mutation, use_query
 
 
 @idom.component
 def hello_world():
-    return idom.html.h1({"id": "hello-world"}, "Hello World!")
+    return html.h1({"id": "hello-world"}, "Hello World!")
 
 
 @idom.component
 def button():
     count, set_count = idom.hooks.use_state(0)
-    return idom.html.div(
-        idom.html.button(
+    return html.div(
+        html.button(
```

Comment:
Let's leave `idom.html` -> `html` changes for the `view_to_component` branch, just to reduce merging complexity.